src/Client.php

```
@@ -569,4 +569,136 @@ function (array $user) {
             null
         );
     }
+
+    public function addUsersToAccount(array $arguments): CallResult
+    {
+        $payload = [
+            "account" => $this->accountIdentifiersToArray(
+                new AccountIdentified(
+                    isset($arguments["account"]["accountId"]) ? (string) $arguments["account"]["accountId"] : null,
+                    isset($arguments["account"]["domain"]) ? (string) $arguments["account"]["domain"] : null
+                )
+            ),
+            "users" => array_map(
+                function (array $user) {
+                    return $this->userIdentifiersToArray(
+                        new UserIdentified(
+                            isset($user["userId"]) ? (string) $user["userId"] : null,
+                            isset($user["email"]) ? (string) $user["email"] : null
+                        )
+                    );
+                },
+                array_values($arguments["users"])
```

Maybe throw error if users key is not set?

`array_key_exists` or `isset`